arrows/matlab/mxtypes.h

```diff
@@ -39,14 +38,16 @@ namespace kwiver {
 namespace arrows {
 namespace matlab {
 
-/************************************************************/
-/* Traits for fundamental datatypes.
-   Don't use with function templates due to type promotion!
-   (Ignore it and get PITA!)                                */
-/************************************************************/
+//************************************************************
+///
+/// Traits for fundamental datatypes.
+/// Don't use with function templates due to type promotion!
+/// (Ignore it and get PITA!)
+///
+//************************************************************
```

Comment:
Again, here, this probably should be manually adjusted. It's also *actually* not a Doxygen comment.

Reply:
Fixed.